terraform/modules/alertmanager/templates/alertmanager.tpl

```diff
@@ -9,6 +9,7 @@ global:
 
 templates:
 - '/etc/alertmanager/default.tmpl'
+- '/etc/alertmanager/notify.tmpl'
```

Comment:
we'll either need to add this into `data.template_file.alertmanager_nlb_container_defn` next to `default.tmpl`, or (probably easier for now) whack it into `default.tmpl`.